code/bit-manipulation/Minimum bits to flip such that XOR of A and B equal to C

```diff
@@ -0,0 +1,33 @@
+// C++ code to count the Minimum bits in A and B
```

Comment:
This file needs a file extension, needs to be shorter, and also needs to get rid of spaces.